Makefile.core.mk

```diff
@@ -125,12 +132,15 @@ foo2:
 
 .PHONY: init
 init:
+	@echo "ISTIO_SHA = ${ISTIO_SHA}"
+	@echo "HUB = ${HUB}"
+	@echo "TAG = ${TAG}"
 	bin/init.sh
 
 include tests/tests.mk
 
 # make lint-yaml seems to fail with pipefail, so remove now.
```

Comment:
please file a bug and assign to me. Not super satisfied with the workaround in: https://github.com/istio/common-files/pull/200

Reply:
I don't seem able to create issues on https://github.com/istio/common-files/ ... permissions thing?

Reply:
Sure thing, @sdake  ... https://github.com/istio/istio/issues/21854